### PR TITLE
Support steal-conditional in optimized builds

### PR DESCRIPTION
**Pull request by m-mujica**
https://github.com/stealjs/steal-conditional/issues/43

- [ ] - Once steal-tools PR passed, merge and release steal-conditional changes
- [ ] - Replace `steal-conditional` path in package.json to its latest release
- [ ] - Merge this PR and release.

### CODE REVIEW

File: lib/graph/make_graph.js [hunk 1]
```diff
@@ -34,16 +34,41 @@ module.exports = function(config, options){
...
+				enumerable: true
+			},
+			identifiersToResolve: {
+				value: [],
+				enumerable: true
+			}
+		});
+
+		localSteal.System.normalizeMap = {};
+		localSteal.System.slimConfig = slimLoaderConfig;
```
Comment (m-mujica) [inline lib/graph/make_graph.js]: I first thought about hiding these being some getter/setters, but I was not sure about the API and ended up just using POJOs. Thoughts?

**Comment by matthewp** [comment]
👍 Looks good, will release tomorrow morning.

**Comment by matthewp** [comment]
Reminder to myself: release https://github.com/stealjs/steal-conditional first and update this PR.

**Comment by m-mujica** [comment]
> 👍  Looks good, will release tomorrow morning.

@matthewp this should be in a minor release, right? It's technically not a bug fix.

**Pull request merged**